tests/test_opera_manager.py

```diff
@@ -0,0 +1,80 @@
+import os
+
+import pytest
+from selenium import webdriver
+
+from tests.test_cache import cache, delete_cache
+from webdriver_manager.driver import OperaDriver
+from webdriver_manager.opera import OperaDriverManager
+
+PATH = '.'
+
+
+def delete_old_install(path=None):
+    if not path is None:
+        path = os.path.abspath(path)
+        try:
+            os.remove(os.path.join(os.path.dirname(path))) # maybe change this
+        except:
+            pass
+
+
+def test_operachromium_manager_with_correct_version():
+    driver_path = OperaDriverManager("v.2.45").install()
+    assert os.path.exists(driver_path)
+
+
+def test_operachromium_manager_with_selenium():

```

Comment:
dummy test?

Reply:
I copied the firefox test and test_operachromium_manager_with_selenium only work with a opera binary
I guessed that you didn't have opera install on Travis-CI
I had 6 test pass on my machine

Reply:
Yes! So we have to add opera installation in travis

Reply:
The browser itself not the operachromiumdriver
Maybe you can do it with https://saucelabs.com/
https://docs.travis-ci.com/user/gui-and-headless-browsers/
i don't know much about CI and pytest